src/main.zig

```diff
@@ -48,9 +48,18 @@ fn loop(
     var buffered_writer = std.io.bufferedWriter(std_out);
     const writer = buffered_writer.writer();
 
+    var arena = std.heap.ArenaAllocator.init(std.heap.page_allocator);
+    defer arena.deinit();
+
     while (true) {
-        var arena = std.heap.ArenaAllocator.init(server.allocator);
-        defer arena.deinit();
+        defer {
+            // Mom, can we have garbage collection at home?
```

Comment:
This hits too close to home.
```suggestion
            // Mom, can we have garbage collection?
```